__tests__/app.test.js

```diff
@@ -128,6 +128,7 @@ describe("/api/articles", () => {
             created_at: expect.any(String),
             votes: expect.any(Number),
             article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
```

Comment:
can we have a new test for this instead?
so it has comment count as well as everything else